Add sponsorship policy. [WIP] (#128)
NixOS/foundation: Sponsorship.md

# Sponsorship

1. Purpose:
This policy aims to facilitate clear, prompt, and equitable determinations of proposed sponsorships without necessitating extensive debate or uncertainty.

> Nonetheless, the Foundation retains the right to decline sponsorship through a Board vote in cases where it would impede the Foundation's mission.

Reviewer: ```suggestion
This policy is also established to protect the NixOS project from bad publicity which may occur because of sponsorships.
Therefore, the Foundation retains the right to decline sponsorship through a Board vote in cases where it would impede the Foundation's mission.
```

This is why we are doing all of this in the first place, so let's be frank about it.

Reviewer 2: Under the table decisions in the name of "protecting the NixOS project" will be more embarrassing to find out than any potential damage a sponsor could do.

Author: I believe this is resolved as of yesterday's call.

2. Scope:
This policy applies to all sponsorships at official NixOS events.
This encompasses event sponsors as well as companies delivering talks, hosting booths, etc.
Additionally, this policy extends to sponsorship messages within Nixpkgs itself or other official resources such as documentation and the homepage.

3. Rules for Sponsors:
The list of rules may be expanded or modified in the future as necessary to reflect the current decision-making process.

    1. Sponsors must be individuals or organizations that are legally able to operate in compliance with the laws and policies of their own origin, that of the NixOS Foundation, and of relevant third-party agreements.
    2. Sponsorship links, materials, and content must be suitable for minors and work environments and should align with the Foundation's mission.
    3. Event organizers may impose additional rules for event sponsorships, such as setting a theme or requiring a specific local affiliation.

4. Procedures

>     1. The Sponsor Chair will submit sponsorship proposals to the Selection Committee via email, including the event organizers and the Foundation Board.

Reviewer 3: ```suggestion
    1. The Sponsor Chair will submit sponsorship proposals to the Selection Committee via email, composed of the event organizers and the Foundation Board.
```
perhaps clearer?

>     1. The Sponsor Chair will submit sponsorship proposals to the Selection Committee via email, composed of the event organizers and the Foundation Board.

Reviewer: ```suggestion
    1. The Sponsor Chair will submit sponsorship proposals to the Selection Committee via email, which is composed of the event organizers, the Foundation Board, the marketing team and the moderation team.
```

Reviewer 4: I like this. One concern is how much overhead this new process will add. Organizing a conference is already hard enough, especially with all the timings. So keeping this to a minimum would be helpful. In that spirit, having the foundation involved means the overhead for decision-making can be reduced.

Reviewer 3: @piegamesde What is the rationale to include each of these in selection committee?
- Marketing?
- Moderation?

The clearest rationale given was by uninsange: "per its duty, the Moderation team is necessarily exposed to a fairly diverse sampling of the NixOS community". I'm going to reject this; they are exposed to the most divisive sampling of the community, and there is plenty of precedent that Moderation teams should not involve themselves in contentious topics while moderating due to conflicts of interest. [Example](https://rust-lang.github.io/rfcs/1068-rust-governance.html?highlight=moderation#moderation-team)

I don't see why Marketing should have a veto (disclaimer: I am on the Marketing team). While we may have some insight into the broader public sentiment, I would not want to claim this is strong enough to mean we know better than a local event team. I am extremely hesitant to claim that we have some special insight into the acceptable morals+ethics that should give us a veto.

The more groups that are involved leads to more discussion and debate. The Moderation Team has decided that its decisions are not up for public debate; people should contact the team directly or use a few limited channels to engage in a civil manner. I don't think the event teams should declare a similar policy, but why shouldn't they?  Why should only the event teams be subject to veto by other teams? Will each official team's decisions be up for review/debate/veto by other teams? Will Moderation establish a similar set of transparency policies and allow a veto other teams to oversee their actions? This way leads to madness and division. We are establishing a precedent for how our organization is run. I've looked around and talked to many other communities - I spent an entire day at SCALE talking to a wide array of communities about their policies - and I've not found a single example of such a veto being given to a Moderation nor a Marketing team.

(@zimbatm and others) I was quite impressed with how Postgres handles their community (https://momjian.us/main/blogs/pgblog/2023.html#September_11_2023). Their approach is to ensure transparency, make sure people are not tricked, but otherwise let the teams do their work. Please read through their approach and [policies](https://www.postgresql.org/about/policies/) - and consider if we should adopt this general approach to our community organization.

Reviewer 5: I will be opinionated because I'm running out of energy, I will speak for myself as a member of the project lead team of NixCon, but I am not comfortable any future NixCon calling themselves as such if they don't have the **moderation team** in their **selection committee** given the current circumstances that made us discuss all of this.

You can interpret this as my own veto as a PL member. As NixCon are the most important events that the Foundation organize, this sponsorship policy is most likely related to NixCons, obviously, someone can use another name and desire to do adopt another policy with another PL team. I would dare to say this would not be an effective way to evade my prior "highly suggested" recommendation on having the moderation team.

If the moderation team is not part of the decision-making, it's highly likely that the next mistake will engulf again the community and will cause many strained folks in the ecosystem. Thus, I don't see how any large, serious event with sponsorship can escape from having the moderation team in the loop as stakeholders.

If the moderation team is deemed problematic, that's a problem for another venue, albeit it's related to this one. We should be frank on this too, if some folks believe the moderation team to be problematic, they should raise it appropriately (read: not like some folks did in the days surrounding the situation that created the need for this document), ultimately, if there's a need to replace the moderation team, that shall be done with the consequences this can bear with respect to the community trust and the active contributors can decide for themselves whether they want to spend their free time in such an ecosystem after all decisions are done. Sorry to be brutal, but I hate going around in circles.

Reviewer 6: This was already discussed during the open board call, and we agreed to have both the moderation and the marketing team in here for a first version. Everyone is welcome to re-discuss it, but _after_ we have this first iteration.

Reviewer 7: Would it be within scope for this document to also mention what is within the scope/"What counts as" a formal NixOS event?

Author: It appears imperative to convene another meeting to delve into several pertinent matters that remain open for discussion. Specifically, we should address:

- Fine-tuning our word choices to ensure precision and clarity.
- The criteria for selecting the sponsor chair.
- Establishing a protocol for resolving objections effectively.
- Implementing mechanisms for transparently documenting objections.
- Reassessing the composition of teams/groups comprising the selection committee.

By addressing these points, we can ensure a more comprehensive understanding and facilitate smoother decision-making processes.

Reviewer 5: We can pass a first chunk of policy without resolving all the first points except the composition of teams/groups compromising the selection committee.

We could technically even do it in multiple steps, let the selection committee undefined, pass the rest of the text, then work on the definition of the selection committee, etc.

Furthermore, we have two/three ways to proceed about that:
(a) We schedule a meeting urgently and try to involve the folks participating in that discussion, ideally before next Tuesday.
(b) We resolve between ourselves the remaining objections before next Tuesday ideally
(c) We make the Foundation board resolve the remaining objections

If people want (a), I would like to know it quickly and if we cannot get this meeting going before next Tuesday, we should carefully consider (b) or (c) and if we really cannot consider it, I can post to postpone again the decision in the next 2 weeks or by the next week if we do exceptionally a board meeting to decide.

> @refroni Would it be within scope for this document to also mention what is within the scope/"What counts as" a formal NixOS event?

I would personally put it like this: any event with a large audience (>= 50/100) that uses the Nix brand.

We can weaken, for starters, by saying: any event with a large audience that has a financial relationship with the NixOS Foundation.

Author: > We can pass a first chunk of policy without resolving all the first points except the composition of teams/groups compromising the selection committee.
> 
> We could technically even do it in multiple steps, let the selection committee undefined, pass the rest of the text, then work on the definition of the selection committee, etc.

While I agree with the approach of passing the core aspects of the policy first and addressing other points subsequently, I share a concern regarding potential loss of momentum if we delay resolution on certain elements.

> (a) We schedule a meeting urgently and try to involve the folks participating in that discussion, ideally before next Tuesday.

Could you provide further context on the urgency of completing this before next Tuesday? Is it tied to an announcement on Discourse or another critical timeline?

If we opt for (a), and it aligns with the timing of our last board meeting, I'm available over the next few days, including the weekend, if needed, to accommodate scheduling.

Reviewer 5: > While I agree with the approach of passing the core aspects of the policy first and addressing other points subsequently, I share a concern regarding potential loss of momentum if we delay resolution on certain elements.

I may be wrong, but my feeling is that we are not going to run out of momentum given the intensity of the engagement, though we will exhaust ourselves by trying to get agreement on everything right now and this will cause instantaneous disengagement, more on that in the next paragraph.

> Could you provide further context on the urgency of completing this before next Tuesday? Is it tied to an announcement on Discourse or another critical timeline?

Yes, the longer we delay passing at least a first "policy" even incomplete in terms of the exact details, the more people are tired out / burned out and are leaving the project, at the time of speaking, we lost another person who is handling a significant part of operations in nixpkgs over dismissive comments like https://github.com/NixOS/foundation/pull/128#issuecomment-2035356976.

My personal priority is to get the community in a working state again before we end up losing so much that nixpkgs is set back for years.

> If we opt for (a), and it aligns with the timing of our last board meeting, I'm available over the next few days, including the weekend, if needed, to accommodate scheduling.

cc @Janik-Haag @piegamesde @tomberek @paperdigits as the folks who were vocal on the ongoing discussion, I believe.
Let's create tentatively a slot for that meeting, please use: https://meet.dgnum.eu/ to create a scheduling.

Author: > > 
> 
> I may be wrong, but my feeling is that we are not going to run out of momentum given the intensity of the engagement, though we will exhaust ourselves by trying to get agreement on everything right now and this will cause instantaneous disengagement, more on that in the next paragraph.

That's true. I didn't mention it in my previous comment but the other thing to consider is burn out.

Author: > Let's create tentatively a slot for that meeting, please use: https://meet.dgnum.eu/ to create a scheduling.

I've started an instance here with my availability.
https://meet.dgnum.eu/nix-policy-discussion-874929

@Janik-Haag @piegamesde @tomberek @paperdigits

Reviewer 5: @piegamesde won't be able to attend from what I understand.

Reviewer 5: @djacu @paperdigits @tomberek @Janik-Haag 7, April 2024 at 7PM CEST: https://files.lahfa.xyz/sponsorship-discussion.ics for the Calendar invite.

Author: I believe this is resolved as of yesterday's call.

>     2. Selection Committee members shall be notified at least 2 weeks prior to the event and provided with at least 1 week to submit objections to the Foundation Board via email.

Reviewer: Why submit objections to the Foundation Board instead of the sponsorship chair, especially since the Foundation Board essentially *is* the Selection Committee?

Reviewer 6: A practical reason in that there's no way to reach out to the committee in general, while there's an email address to the foundation.
I'm fine with relaxing it if it doesn't make thing s too ambiguous:

```suggestion
    2. Selection Committee members shall be notified at least 2 weeks prior to the event and provided with at least 1 week to submit objections to committee members in written form.
```

Reviewer 8: > 2. Selection Committee members shall be notified at least 2 weeks prior to the event and provided with at least 1 week to submit objections to the Foundation Board via email.

So if a person in the community who is not part of selection committee wants to weigh in on a sponsor or potential sponsor, how would that be done? 

Right now it seems like the prevailing perception is that the moderation team will raise concerns on behalf of the community. But the decision making on what constitutes raising an objection is not documented. And the decision making for each team is also not a documented process and should be left opaquely up to each team?

Reviewer: > So if a person in the community who is not part of selection committee wants to weigh in on a sponsor or potential sponsor, how would that be done?

They can't because sponsorships will unlikely to be known until finalized. I initially proposed to have the proposals open for community discussion in some form, however this was rejected

    3. Objections should be specific to the sponsor and cite relevant reasons why the sponsorship would impede the NixOS mission.

>     If at least two (2) objections are received, the sponsor will be subjected to a general vote by the Selection Committee.
> 
>     4. A sponsor will be accepted unless:
> 
>         1. A majority (>50%) of the event organizers vote to decline the sponsor, or
>         2. A majority (>50%) of the Foundation Board vote to decline the sponsor.

Reviewer: ```suggestion
    If at least one team of which the selection committee is composed of raises objections for a sponsor, then these objections must be discussed with the selection committee. If after that discussion the objection is not retracted, then the sponsor has to be declined. 
```

Some general thoughts about this

- I think it is better to think in terms of teams as units with their own internal decision making process rather than of its individual members
- I think that declaring voting mechanisms in such detail may not be necessary; I'd like to avoid over-specifying procedures where possible. Moreover, I'd prefer a veto-based mechanism with lazy consensus over a voting-based mechanism.

Author: I disagree with this change:

- Allowing objections from individual teams without a clear threshold for validity could open the door for manipulation or strategic objections aimed at excluding certain sponsors for reasons unrelated to the sponsor's suitability.
- With the suggested changes, it would only require that one team raise an objection and refuse to retract it.

Reviewer 2: I agree with @djacu. This policy should seek to minimize the possibility of any under the table decisions coming from _any_ team. From an outsider perspective, the moderation team has a lot of influence. Using community outrage as a proxy for decisionmaking is a sure way to remove any illusion that a decision was made neutrally.

Reviewer 4: I like @piegamesde's approach better, as it leaves more room for conversation. Ideally, the decision is also put in context with the funds we need to run the event, what sponsors have become available, and how it fits with the NixCon brand.

We can always adjust the policy if any issues come from it.

Reviewer 2: The main concern I have to keep that trust is avoiding accidental "black box" decision-making on this stuff. We go to some serious efforts for full transparency with nixpkgs itself. I think our community taking the same approach is a good direction, and to that end the policy ought to be "boring." If any questions arise about a sponsor, we just go through the checklist, and it's plausible (based on the design of the rules) that no one had conflicts of interest at any step.

Author: > I like @piegamesde's approach better, as it leaves more room for conversation. Ideally, the decision is also put in context with the funds we need to run the event, what sponsors have become available, and how it fits with the NixCon brand.
> 
> We can always adjust the policy if any issues come from it. In general, we should operate on the basis of trust.

Section 4.4 addresses resolution rather than conversation. The policy does not explicitly outline guidelines for conversation, as it is deemed appropriate for the selection committee/teams to determine. Given its role as the final authoritative decision, it should be crafted to minimize potential biases, partiality, and manipulation.

Author: I believe this is resolved as of yesterday's call.

>     1. The Sponsor Chair will submit sponsorship proposals to the Selection Committee via email, composed of the event organizers and the Foundation Board.
> 
>     2. Selection Committee members shall be notified at least 2 weeks prior to the event and provided with at least 1 week to submit objections to committee members in written form.
> 
>     3. Objections should be specific to the sponsor and cite relevant reasons why the sponsorship would impede the NixOS mission.
>     If at least two (2) objections are received, the sponsor will be subjected to a general vote by the Selection Committee.
> 
>     4. A sponsor will be accepted unless:
> 
>         1. A majority (>50%) of the event organizers vote to decline the sponsor, or
>         2. A majority (>50%) of the Foundation Board vote to decline the sponsor.

Reviewer 9: As a compromise between the efficiency of fewer decisionmakers and slimmer processes, and the acknowledged stake that Moderation and Marketing have in the outcome of sponsorship, here's an idea:

```suggestion
    1. The Selection Committee is composed of the event organizers and the Foundation Board.

    2. The Sponsor Chair will submit sponsorship proposals to the Selection Committee via email.

    3. The Selection Committee shall be notified at least 2 weeks prior to the event and provided with at least 1 week to submit objections to committee members in written form.

    4. The Selection Committee should consider objections that are specific to the sponsor and relevant to the NixOS mission, consulting with the Marketing team and the Moderation team as necessary.

    5. Either the event organizers or the Foundation board may decline the sponsor.
```

My basic reasoning here is that we should trust the Foundation Board to consult the people who may be impacted and use their veto power on their behalf. If we didn't think the Foundation would act in the best interests of Nix, we might as well pack up our bags, because what would the point of any of this be after that?

Reviewer 5: We already have tried this, this resulted in the current situation. The Foundation board has many things to do, this is about giving a mandate to the people who are the closest to the community to handle this.

The Foundation act in the best interests of Nix by delegating this to the group that also aact in the best interests of Nix, for example.

Reviewer 10: The moderation team is not "the closest to the community".

Reviewer 5: Can you cite another set of contributors who are well positioned to make those calls?

Reviewer 10: The board is best positioned to make these kinds of executive decisions. That's their job.

Reviewer 3: That looks like it is more accurate as well. The earlier one is a bit over-specific on the details in policy regime where we don't generally have that much formal procedure.

Reviewer 11: > The board is best positioned to make these kinds of executive decisions. That's their job.

The board is stuffed with people who have [commercial interest](https://github.com/NixOS/foundation?tab=readme-ov-file#board-affiliation-transparency) in Nix/Nixpkgs/NixOS. You believing they are  “closest to the community” reveals you to be an outsider to this community.

Reviewer 12: > The board is stuffed with people who have [commercial interest](https://github.com/NixOS/foundation?tab=readme-ov-file#board-affiliation-transparency) in Nix/Nixpkgs/NixOS

Although true, I would also say that would be conjunction with massive community legacy:
- Eelco Dolstra: The creator of Nix, 20+ years of contributions
- Théophane Hufschmitt: Contributor of 5+ years, member of Nix maintenance team
- Jonas Chevalier: Contributor of ~9+ years, pillar of the community, creator of Direnv.
- Domen (recently left): Contributor of 10+ years, ~6x NixOS release manager. Creator of Cachix.
- Ron Efroni - Sadly not personally familiar with him. Only one without a large Nix legacy.

Reviewer 12: I think "the board is stuffed pillars of the community and also want to see Nix be a commercial success" is a better way to phrase that.

Reviewer 11: There is no question they have had a big influence on the community in the past, but nowadays, they rarely interact with or contribute to nixpkgs anymore. But that is precisely where the largest part of the community interaction actually happens.

If you're not there you're likely missing out on a lot of people that keep nixpkgs running. And it is exactly those people who are in need of representation here.

Reviewer 12: I agree, nixpkgs is usually the gateway drug. And it's a "choose your own adventure" from there. I would still consider work on NixOS/nix as contributions though.

Grahamc was Mr. Infra + Mr. moderation + Mr. ofBorg for many year. I would still consider his contributions to be just as valuable as well. <3 you @grahamc. Glad his hats were distributed amongst many.

1. Purpose:
This policy aims to facilitate clear, prompt, and equitable determinations of proposed sponsorships without necessitating extensive debate or uncertainty.

2. Scope:
This policy applies to all sponsorships at official NixCon events.
This encompasses event sponsors as well as companies delivering talks, hosting booths, etc.
Additionally, this policy extends to sponsorship messages within Nixpkgs itself or other official resources such as documentation and the homepage.

3. Rules for Sponsors:
The list of rules may be expanded or modified in the future as necessary to reflect the current decision-making process.

    1. Sponsors must be individuals or organizations that are legally able to operate in compliance with the laws and policies of their own origin, that of the NixOS Foundation, and of relevant third-party agreements.
    2. Sponsorship links, materials, and content must be suitable for minors and work environments and should align with the Foundation's mission.
    3. Event organizers may impose additional rules for event sponsorships, such as setting a theme or requiring a specific local affiliation.

4. The Selection Committee will be composed of:

>     - 2 representatives from the event organizers

Reviewer 13: ```suggestion
    - 2 representatives from the event organizers, including 1 member of the NixCon PL team if the PL team is engaging on the event
```

Reviewer 4: One issue raised is that the PL team isn't officially defined yet (even if it exists and is doing a great job).

To solve this and move forward, I proposed creating a second document that establishes the PL team. As part of it, the PL team gets one of the event organizers' votes on all the events they are responsible for.

Reviewer 5: What do you think of this informal post and definition (I must assume) https://discourse.nixos.org/t/governance-through-project-leads-in-nixcon-2024/33981 ? What should be improved to make it more defined ?

Reviewer 4: I like it overall. It does a good job of defining the scope, the roles and responsibilities, the team composition, and how to contact you.

One unclear point was the distinction between Bootstrap PL and PL, as well as the mechanisms around it. Are the PLs the event organizers for a specific year? This must be better defined before it can be referenced in this document.

Another aspect could be to officially assign the NixCon brand and assets to the PL. In that scenario, the foundation wouldn't even have to be involved creating this very policy, as the PL would be empowered to write it themselves.

Then give the team a homepage, move this to a founding document, done.

Reviewer 5: Here we go. https://github.com/NixOS/foundation/pull/134

Reviewer 4: Ok, we can work on this separately then, and resolve this thread.

Don't forget to add this notion in the other document: that the PL team would take one vote of the events team.

Reviewer 11: Including this here would IMO be the sensible thing to do, given that this was the compromise that led to the agreement on the call.

Reviewer 13: No mention of the PL team in the sponsorship policy is a deal-breaker for me.

    - 1 representative from the Foundation Board
    - 1 representative from the Moderation team
    - 1 representative from the Marketing team

5. Procedures

    1. The event organizers will submit sponsorship proposals to the Selection Committee via email.

    2. Selection Committee members shall be notified at least 2 weeks prior to the event and provided with at least 1 week to submit objections to committee members in written form.

    3. Objections should be specific to the sponsor and cite relevant reasons why the sponsorship would impede the NixOS mission.

    4. If a consensus cannot be reached through discussion, the objection shall be brought to a vote.

>     The sponsor will be accepted if a strong majority of the committee votes to accept the sponsor.

Reviewer 13: ```suggestion
    The sponsor will be accepted if a strong majority (at least 4 votes in favor) of the committee in reached to accept the sponsor.
```

Author: Pushed a couple updates. One to use more precision language and one to address this. We good?

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Reviewer 13: I want the language to be about committee members and not expressed votes. I'm fine if we say something like "A sponsor will be accepted if and only if 70% or more of committee members are in favor."

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Reviewer 13: I don't see why we don't keep the clear "4/5 members should be in favor". When we revisit the committee composition, we have to revisit this document anyway.

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Reviewer 13: ```suggestion
    A sponsor will be accepted if and only if at least 4 members of the committee are in favor.
```

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Author: @JulienMalka I'm not sure what is missing. What you have suggested is pretty close to the current language is it not? Or are you just stating what you would like to see and not making a suggestion?

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Author: > @JulienMalka I'm not sure what is missing. What you have suggested is pretty close to the current language is it not? Or are you just stating what you would like to see and not making a suggestion?

Ah sorry your second and third comment hadn't loaded yet when I commented. All clear now

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Reviewer 13: > @JulienMalka I'm not sure what is missing. What you have suggested is pretty close to the current language is it not? Or are you just stating what you would like to see and not making a suggestion?

1) The `if and only if` makes it clear that the only way for a sponsors to get approved is through the committee agreement. It's important.
2) I want the language to be clear that we need a majority of members and not voters (abstention counts as a no). It's also important.
3) I think that 4/5 is more clear than 70% but I can compromise on that.

>     4. In the event of an impasse during deliberation, objections shall be put to a vote.
>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Author: To preserve the understanding of what a "significant/strong/super" majority is in the future, how about the following?

```suggestion
    4. In the event of an impasse during deliberation, objections shall be put to a vote.
    A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.
    With the current composition of the selection committee, acceptance necessitates at least 4 members in favor.
```

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Author: To preserve the understanding of what a "significant/strong/super" majority is in the future, how about the following?

```suggestion
    4. In the event of an impasse during deliberation, objections shall be put to a vote.
    A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.
    With the current composition of the selection committee, acceptance necessitates at least 4 members in favor.
```

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Reviewer 13: Can we change `70% of the committee votes` to `70% of the committee members` ? And for the rest no strong opinion on adding or not the last sentence.

>     A sponsor will be accepted if a significant majority, defined as greater than 70% of the committee votes, is in favor of acceptance.

Author: > Can we change `70% of the committee votes` to `70% of the committee members` ? And for the rest no strong opinion on adding or not the last sentence.

Ah I understand what you were trying to convey now. Members not votes. Yeah that's better.
